configs/recognition_audio/avslowfast/README.md

```diff
@@ -0,0 +1,62 @@
+# AVSlowFast
+
+## Introduction
+```
+@article{xiao2020audiovisual,
+  title={Audiovisual SlowFast Networks for Video Recognition},
+  author={Xiao, Fanyi and Lee, Yong Jae and Grauman, Kristen and Malik, Jitendra and Feichtenhofer, Christoph},
+  journal={arXiv preprint arXiv:2001.08740},
+  year={2020}
+}
+```
+
+## Model Zoo
+
+### Kinetics-400
+
+|config | n_fft | gpus | backbone |pretrain| top1 acc/delta| top5 acc/delta | inference_time(video/s) | gpu_mem(M)| ckpt | log| json|
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[avslowfast_r50_32x2x1_239e_kinetics400_audio_feature](/configs/recognition_audio/avslowfast/avslowfast_r50_32x2x1_239e_kinetics400_audio_feature.py)|1024|8| ResNet50 | None |||x||[ckpt]()|[log]()|[json]()|
```

Comment:
Is the ckpt ready?

Reply:
Still training now, takes more than training a slowfast.